bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -483,8 +483,11 @@ def from_api_repr(cls, resource):
         :returns: instance
         """
         name = resource.get('name')
-        array_type = resource['parameterType']['arrayType']
-        values = resource['parameterValue']['arrayValues']
+        array_type = resource['parameterType']['arrayType']['type']
```

Comment:
Is there a better way to do it? Technically it should be recursive since an array could contain a struct, but I didn't want to modify the design too much in this PR. (Structs have a similar problem since they can contain arrays or more structs, but we don't handle that case yet)